<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1051</width>
    <height>679</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>1051</width>
    <height>679</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>251</x>
      <y>540</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>选择文件</string>
    </property>
   </widget>
   <widget class="QComboBox" name="directoryComboBox">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>540</y>
      <width>381</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButton">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>540</y>
      <width>47</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="origin_textEdit">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>441</width>
      <height>461</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="translate_textEdit">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>50</y>
      <width>441</width>
      <height>461</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="translate_btn">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>220</y>
      <width>93</width>
      <height>121</height>
     </rect>
    </property>
    <property name="text">
     <string>开
始
翻
译</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>20</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>原文</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>20</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>结果</string>
    </property>
   </widget>
   <widget class="QPushButton" name="choose_btn">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>570</y>
      <width>93</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>我选好了</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1051</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>帮助</string>
    </property>
    <addaction name="instruction_action"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="instruction_action">
   <property name="text">
    <string>说明</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
